httpsredirect.go

```diff
@@ -1,27 +1,19 @@
 package gokrazy
 
 import (
+	"net"
 	"net/http"
-	"strings"
 )
 
-// TODO: Configurable https-port
-func httpsRedirect(handler http.Handler) func(w http.ResponseWriter, r *http.Request) {
-	return func(w http.ResponseWriter, r *http.Request) {
-		// get hostname without port
-		target := r.URL.Host
-		if port := ":" + r.URL.Port(); strings.HasSuffix(target, port) {
-			target = strings.TrimSuffix(target, port)
-		}
-
-		// serve localhost on http
-		if target == "localhost" {
-			handler.ServeHTTP(w, r)
-			return
-		}
-
-		r.URL.Host = r.Host
-		r.URL.Scheme = "https"
-		http.Redirect(w, r, r.URL.String(), http.StatusFound) // Redirect to https
+func httpsRedirect(w http.ResponseWriter, r *http.Request) {
+	// get hostname without port
```

Comment:
This comment is now obsolete (no hostnames involved)

Reply:
Removed the err check as well since:

> The HTTP server in this package sets RemoteAddr to an "IP:port" address before invoking a handler. This field is ignored by the HTTP client.